collections/_posts/2025-05-06-wcag2mobile.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "For Review: Guidance on Applying WCAG 2.2 to Mobile Applications (WCAG2Mobile) — First Draft Note"
+date: 2025-05-06
+---
+
+[Guidance on Applying WCAG 2.2 to Mobile Applications](https://www.w3.org/TR/wcag2mobile-22/) is available as a first draft Note. WCAG2Mobile describes how Web Content Accessibility Guidelines (WCAG) 2.2 principles, guidelines, and success criteria can be applied to mobile applications, including native mobile apps, mobile web apps, and hybrid apps using web components inside native mobile apps. We welcome your comments on the direction and approach of this in-progress resource. To comment via email or GitHub, see links in the 'Status of This Document' section.
```

Comment:
```suggestion
[Guidance on Applying WCAG 2.2 to Mobile Applications](https://www.w3.org/TR/wcag2mobile-22/) is available as a first Draft Note. WCAG2Mobile describes how Web Content Accessibility Guidelines (WCAG) 2.2 principles, guidelines, and success criteria can be applied to mobile applications, including native mobile apps, mobile web apps, and hybrid apps using web components inside native mobile apps. We welcome your comments on the direction and approach of this in-progress resource. To comment via email or GitHub, see links in the 'Status of This Document' section.
```

Since a [W3C Draft Note](https://www.w3.org/standards/types/#x2-5-1-draft-note) is a specific type of document, I think the uppercase D is necessary.